2026-cfp.md

```diff
@@ -0,0 +1,75 @@
+
+We invite you to [participate](https://fosdem.org/submit) by presenting your work and projects in this space. We are primarily looking for 20 minutes presentations and also welcome short lightning talks. Please submit them at TBD.
+
+Once again, this will be an excellent opportunity to show what you are doing and discuss it with like-minded people from the community.
+
+> [!NOTE]
+> [FOSDEM](https://fosdem.org) is Europe's largest gathering of Free and Open Source Software developers and enthusiasts, with thousands of people attending and hundreds of talks taking place each year. FOSDEM 2026 will happen on the weekend of 31 January and 1 February 2026 as an in-person event in Brussels, Belgium at the ULB university.
+
+<figure>
+  <img src="img/fosdem-2023-crowd.jpg" alt="Picture from the devroom in 2023">
+  <figcaption>The Railways and Open Transport devroom was very-well visited</figcaption>
+</figure>
+
+## Key dates
+
+For the Call for Presentations process:
+* 30 November: CfP submission deadline
+* 15 December: Announcement of selected talks
+* 1 February: Railways and Open Transport devroom at FOSDEM, you must be available onsite to present your talk (tentatively between 13:15 and 17:00 local time)
+
+## Topics
+
+We are interested in presentations on any topic related to Free and Open Source Software in the context of railways and public transport. Possible topics include:
+
+* Railways and Heavy Transport: timetabling, signalling, operation, simulation, dispatching, infrastructure planning, network management
+* Information and accessibility: multimodal trip planning (train, bus, metro, ferry, air...), inclusion, open source ticketing, mobility as a service
+* Data Analytics: working with historical data, forecasts, machine learning, AI/AR
+* Cross-cutting themes: international collaborations and open standards (e.g. NeTEx, SIRI, GTFS), climate and sustainability impact, gamification and education, policy
+
+The audience is developers and enthusiasts of Free and Open Source Software. This includes a good number of railway fans and people active in related communities and organizations.
+
+## Submission process
+
+Please submit your proposals via the [FOSDEM's conference system](https://fosdem.org/submit) by 30 November 2025, 23:59 (Europe/Brussels, UTC+1). The account from last year will work. If you still have accounts from FOSDEM editions before 2024, these won't work, please create new ones.
+
+Important notes:
+* Your on-site availability is required, we do not accept remote presentations.
+* Preferred talk length is 20 minutes including Q&A. We also welcome shorter talks. Please state the length of the talk in the submission notes.
+* The conference language is English.
+
+Here is how you can submit a presentation to our devroom:
```

Comment:
I wonder, if we need these instructions in this detail. The conference system should be mostly self-explaining. Maybe we should just say:

```suggestion
Submit your proposal at https://fosdem.org/submit. Make sure to select the "Railways and Open Transport" track and fill out all the information which is needed to describe your talk and all speakers. Note that title, abstract and speaker information will be published on the FOSDEM conference web site if your proposal is accepted.
```

Reply:
Yes, makes totally sense. Made that much easier now.